src/client/App.jsx

```diff
@@ -1,11 +1,96 @@
 import React from 'react';
 
-function App() {
-    return (
-        <div className="App">
-            App
-        </div>
-    );
+// Import components
+import MovieGrid from './components/results/MovieGrid';
+import Filters from './components/filtering/filters';
+import HeaderSearch from './components/sections/header-search';
+import HeaderMovie from './components/sections/header-movie-single';
+import Footer from './components/sections/footer';
+
+
+// utils function to generate random colors
+function getRandomColor() {
+    const letters = '0123456789ABCDEF';
+    let color = '';
+    for (let i = 0; i < 6; i++) {
+        color += letters[Math.floor(Math.random() * 16)];
+    }
+    return color;
+}
+
+function sortByDate(a,b) {
+    if (a.year < b.year) {
```

Comment:
@vitkuz as we discussed `sortByDate` and `sortByRate` have the same functionality but with different props. It should be optimized

Reply:
done